apps/website/src/pages/collaborations.tsx

```diff
@@ -235,13 +236,25 @@ const CollaborationsPage: NextPage = () => {
         />
 
         <Section dark className="pb-12 pt-24">
-          <div className="w-full lg:w-3/5">
+          <div className="w-full lg:w-4/5">
             <Heading>Our Collaborations</Heading>
-            <p className="text-lg">
+            <p className="text-balance text-lg">
               We work with other content creators to educate our combined
               audiences, introducing them to the educational ambassadors at
               Alveus and their conservation missions.
             </p>
+
+            <p className="mt-2 text-balance text-lg">
+              We&apos;ve hosted{" "}
+              <abbr
+                title={`Across ${collaborations.length.toLocaleString()} collaboration streams`}
+              >
```

Comment:
This feels like a misuse of abbr, but it does exactly what I want here?

Reply:
Yeah i think this is fine-ish. If we wanna be even more accessible, relying on the title attribute isn't gonna be the best, but i dont think its a big deal here.